scripts/genebuild/create_annotation_configs.pl

```diff
@@ -76,12 +76,12 @@
   $selected_db = "gb_assembly_registry";
 }
 
-my $assembly_registry = new Bio::EnsEMBL::Analysis::Hive::DBSQL::AssemblyRegistryAdaptor(
-  -host    => $assembly_registry_host,
-  -port    => $assembly_registry_port,
-  -user    => 'ensro',
-  -dbname  => $selected_db
-  );
+#my $assembly_registry = new Bio::EnsEMBL::Analysis::Hive::DBSQL::AssemblyRegistryAdaptor(
+ # -host    => $assembly_registry_host,
+  #-port    => $assembly_registry_port,
+  #-user    => 'ensro',
+  #-dbname  => $selected_db
+  #);
```

Comment:
The commented code must be removed because it doesn't explain the code

Reply:
Changes made as requested